generic-styles/index.less

```diff
@@ -1,3 +1,31 @@
 #main-content {
   @import "./content-baked.less";
 }
+#main-content > .os-glossary-container > dl > dt {
```

Comment:
These can be nested inside the `#main-content {` block above. So something like:

```less
#main-content {
  @import "./content-baked.less";

  > .os-glossary-container > dl > dt {
    font-weight: 700;
  }
  // ...
}
```